kubernetes-services/work-scheduler/bin/push-image

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I think @vinnyinverso added a script that works for either AWS CLI 1 or 2 - we should replace these two scripts with that one modified for this image. This would be another good file to add to the templates directory I mentioned in another comment.

Reply:
Changed to use Vinny's script.